wg-xai/README.md

```diff
@@ -0,0 +1,64 @@
+# AI Explainability Working Group
+
+A Working Group dedicated to establishing (designing, developing and documenting) an explainability framework for ODH developers (based on TrustyAI explainability toolkit). 
+Explanations are useful tools in order to:
+Establish trust in predictive models created / used in ODH, especially when deployed in decision making pipelines, e.g. provide explanations together with predictions to end users to allow humans to take better informed decisions
+Debug predictions made by predictive models created / used in ODH, e.g. detect (portions of) model input that are responsible for counterintuitive predictions
+Get a global understanding of general behavior of predictive models, e.g. which features does a model mostly attend to?
+
+## Stakeholder SIGs
+* [SIG ML Developer Experience](/sig-ml-developer-experience)
+* [SIG ML Ops](/sig-ml-ops)
+
+## Expected WG lifetime
+6 months
+
+## Meetings
+* Working Group Meeting: TBD
+  * [Meeting notes and Agenda] TBD.
+
+## Organizers
+
+* Rebecca Whitworth
+* Tommaso Teofili
+* Rui Vieira
+* Rob Geada
+
+## Contact
+- Slack: [#wg-ai-explainabilityt-framework](https://odh-io.slack.com/archives/C03UFCVFFEY)
```

Comment:
The slack channel is now called `wg-ai-explainability` right?

Reply:
yes, correct. I'll fix that.